simulator-core/src/test/java/com/evbox/everon/ocpp/simulator/StationSimulatorFunctionalTest.java

```diff
@@ -637,6 +638,29 @@ void shouldGetHeartbeatIntervalWithGetVariablesRequest() {
         });
     }
 
+    @Test
+    void shouldSendNotifyReportOnGetBaseReportRequest() {
```

Comment:
This class is already too big. Consider extracting your tests to a separate class

Reply:
i agree but that would entail refactoring mocks setups etc. which is not a part of this story, we have a separate story for refactoring our test code.